index.d.ts

```diff
@@ -1,4 +1,6 @@
 declare module "fhirpath" {
+  export function parse(path: string);
+  export function compile(path: string, model: Model);
```

Comment:
Let's describe returning type of compile - it must be function with params fhirData and context

Reply:
and returning type any[]